workshops/book-dash/meeting-notes/2023-09-05-WG.md

```diff
@@ -0,0 +1,57 @@
+# Book Dash [Provisional] Working Group Meeting 2023-09-05
+
+## Attendees 
+* Arielle Bennett
+* Emma Karoune 
+* (Apologies) Esther Plomp
+* (Apologies) Anne Lee Steele
+* (Apologies) Alex Araujo Alvarez
+
+## Agenda 
+* Working Group vs Planning Committee - governance and set up 
+* Applications for Nov 2023 Planning Committee
+* Activities for next Book Dash - discussion
+* Book Dash Hubs Set Up and Guidance
+
+## Decisions & Actions
+* Set up Working Group & separate Planning Committee in Nov following subsequent Book Dash - recommendation to core delivery team
+* Invite current and previous Planning Committee members to apply to WG in Nov - recommendation to core delivery team 
+* Accept all applications submitted to be part of this Planning Committee - recommendation to core delivery team
+* (Arielle) Upload notes to GitHub repo
+* (Arielle) communicate recommendations to core delivery team 
+
+## Notes
+* Book Dash working group vs planning committee team
+    * Assuming that we would still have RPM and RCM on planning committee, do need insight into operations and what is possible 
```

Comment:
RPM and RCM are part of the planning committee and available to support any operations needed.

Reply:
Just to confirm, our roles are more operational and not strategic right? This is emerging from the core delivery team discussions on governance. It's worth asking what our roles are if we are split between the both the working group and the planning committee.

Reply:
Research Community Manager and Research Project Manager definitely need to be on the planning committee otherwise it's hard to stay on top of what needs to be executed (e.g. Scriberia contracts) and also plan other outreach